numba/unicode.py

```diff
@@ -541,6 +541,72 @@ def find_impl(a, b):
         return find_impl
 
 
+def unicode_rfind_check_type(ty, name):
```

Comment:
This function could go in the scope of `unicode_rfind`? I doubt it'll be used anywhere else?